### ISSUE TITLE
Add option for accessibility standards

### ISSUE THREAD

**Issue by acrawly** [felixarntz/bootstrap-for-contact-form-7#24]
Hey, 

I'm using this (awesome) plugin on a few sites that must adhere to accessibility standards (particularly WCAG 2.0 Level AA) and this plugin's HTML output for the Bootsrap labels are not including the `for` property on the `<label>` element.  

Here is an example of the output of a label from one:

``` html
<label class="control-label col-sm-3">
  Your name
  <span class="required">*</span>
</label>
```

And it should be like:

``` html
<label class="control-label col-sm-3" for="id_of_input_here">
  Your name
  <span class="required">*</span>
</label>
```

**Comment by felixarntz** [comment]
Hey,

thank you for letting me know. This currently happens only in some places. Maybe you can give me an advice here: in case of radio or checkbox groups, multiple IDs are required. To what ID should the label point then? I'm not sure how to deal with this issue.

**Comment by acrawly** [comment]
Hey @felixarntz 

I'm by no means an expert so I went to see how the guys at bootstrap do the markup in their <a href="http://getbootstrap.com/css/#forms-example" target="_blank">examples</a> and they do it as so:

``` html
<div class="checkbox">
  <label>
    <input type="checkbox"> Check me out
  </label>
</div>
```

However when I check that markup with a <a href="http://achecker.ca/checker/index.php" target="_blank">accessiblity validator</a>, it still gives me an error. This markup is valid however:

``` html
<div class="checkbox">
  <label for="checkbox">
    <input type="checkbox" id="checkbox"> Check me out
  </label>
</div>
```

I don't know if this would be good on a screen reader however which is the ultimate goal as WCAG 2.0 is just a "guideline" to help us be accessibility friendly. In any regard I believe if the markup was done like the above, we'd be good. What do you think?

**Comment by felixarntz** [comment]
Hey @acrawly,

I think we had a misunderstanding here. I was referring to checkbox groups (or radio groups) where you have multiple checkboxes for basically one field. The example you outlined above is only a single checkbox. In a group, all the checkboxes need to have different IDs - what do you then set the label for attribute to?

**Comment by acrawly** [comment]
Hey @felixarntz,

Looking at the documentation <a href="http://getbootstrap.com/css/#forms-controls" target="_blank">here</a>. We can do something like this:

``` html
<label class="radio-inline" for="inlineRadio1">
  <input type="radio" name="inlineRadioOptions" id="inlineRadio1" value="option1"> 1
</label>
<label class="radio-inline" for="inlineRadio2">
  <input type="radio" name="inlineRadioOptions" id="inlineRadio2" value="option2"> 2
</label>
<label class="radio-inline" for="inlineRadio3">
  <input type="radio" name="inlineRadioOptions" id="inlineRadio3" value="option3"> 3
</label>
```

So you'd basically just have a `<label>` element for each option. 

Note: I still have no idea of if this would look good on a screenreader, it just passes the validator mentioned above.

**Comment by felixarntz** [comment]
But there is also a general label for the group of checkboxes/radio, this label somehow has to refer to the whole group.

**Comment by acrawly** [comment]
Hey @felixarntz,

Sorry about my tardiness, Merry Christmas! 

Okay so I did more research and found a great site, <a href="http://webaim.org/techniques/forms/controls" target="_blank">webaim.org</a>. This is the correct way to present radio buttons and checkboxes:

``` html
<fieldset>
  <legend>Choose a shipping method:</legend>
  <input id="overnight" type="radio" name="shipping" value="overnight">
  <label for="overnight">Overnight</label><br>
  <input id="twoday" type="radio" name="shipping" value="twoday">
  <label for="twoday">Two day</label><br>
  <input id="ground" type="radio" name="shipping" value="ground">
  <label for="ground">Ground</label>
</fieldset>
```

Of course the above isn't Bootstrap markup but we could add the various bootstrap div's and classes within the field set and it should be fine. 

I'm really not very good at Wordpress development but I could make a branch and give this a go?

-Andrew

**Comment by acrawly** [comment]
After cloning this repo and doing some work I found that it already has a major portion of the functionality I'm looking for. With the exception of radio buttons and checkboxes, if I add `id:<id of field here>` to the field short code within Contact Form 7, this generates the label with the `for="<id of field here"` properly. 

If I add the id to a group of checkboxes of `testbox`, it will make the ids of the options `testbox1`, `testbox2`, etc and the label will be in such a format `<label class="checkbox-inline"><input .... /></label>`. I'm moving forward with attempting to change the code to generate `for=<id here>` on the label elements.

**Comment by felixarntz** [comment]
@acrawly Ah yeah, that's it! Thank you, I appreciate it!

**Comment by acrawly** [comment]
@felixarntz I did a pull request to put in the fixes for checkboxes and radio buttons. I don't think I setup the versioning properly for deployment but let me know.

**Comment by felixarntz** [comment]
@acrawly sorry, I have been busy - pull-request looks good, I'm gonna merge it in as soon as I'm ready to take care of the next version release. Probably some time this coming week! :)

**Comment by felixarntz** [comment]
Merged it in! I applied some additional fixes and am going to release the new version in a few minutes! Thanks again for your help! :)
